LeaderboardBackend/Services/IRunService.cs

```diff
@@ -9,8 +9,12 @@ namespace LeaderboardBackend.Services;
 public interface IRunService
 {
     Task<Run?> GetRun(Guid id);
+    Task<GetRunsForCategoryResult> GetRunsForCategory(long id, Page page, bool includeDeleted = false);
     Task<CreateRunResult> CreateRun(User user, Category category, CreateRunRequest request);
 }
 
 [GenerateOneOf]
 public partial class CreateRunResult : OneOfBase<Run, BadRole, BadRunType>;
+
+[GenerateOneOf]
+public partial class GetRunsForCategoryResult : OneOfBase<ListResult<Run>, NotFound, AlreadyDeleted>;
```

Comment:
I don't think it makes sense to use `AlreadyDeleted` here because we're not trying to delete anything.